spec/controllers/posts_controller_spec.rb

```diff
@@ -1,11 +1,10 @@
 require 'spec_helper'
-
 describe PostsController, type: :controller do
   include_context 'JSON API headers'
 
   before(:all) { FactoryGirl.create_list(:post, 3) }
 
-  let(:fields)        { (PostResource.fetchable_fields - %i(id author)).map(&:to_s) }
+  let(:fields)        { (PostResource.updatable_fields - %i(author)).map(&:to_s) }
```

Comment:
`.fetchable_fields` is no longer a class method that returns fetchable fields. I believe `updatable_fields` or `creatable_fields` will both work equally well here?